src/python/pants/backend/python/util_rules/pex.py

```diff
@@ -852,6 +852,7 @@ class VenvPex:
     pex: Script
     python: Script
     bin: FrozenDict[str, Script]
+    venv_rel_dir: str
```

Comment:
It's probably good to tack on ~ `# for tests` to be friendly to future refactorers.